libqtile/widget/memory.py

```diff
@@ -0,0 +1,32 @@
+import re
+from . import base
+
+
+def get_meminfo():
+    val = {}
+    with open('/proc/meminfo') as file:
+        for line in file:
+            key, tail = line.split(':')
+            uv = tail.split()
+            val[key] = int(int(uv[0]) / 1000)
```

Comment:
You can do `int(uv[0]) // 1000` if you do a `from __future__ import division` at the top of the file